tsup.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+	clean: true,
+	bundle: false,
+	dts: false,
+	entry: ["src/**/*.ts", "src/config.json"],
+	format: ["cjs"],
+	minify: false,
+	tsconfig: "tsconfig.json",
+	target: "esnext",
```

Comment:
We should explicitly define this.
```suggestion
	target: "node16.14",
```